tracee-ebpf/main_test.go

```diff
@@ -191,6 +191,18 @@ func TestPrepareFilter(t *testing.T) {
 			expectedFilter: tracee.Filter{},
 			expectedError:  errors.New("invalid retval filter format open.retvall"),
 		},
+		{
+			testName:       "invalid wildcard",
+			filters:        []string{"event=blah*"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid event to trace: blah*"),
+		},
+		{
+			testName:       "invalid not wildcard",
+			filters:        []string{"event!=blah*"},
+			expectedFilter: tracee.Filter{},
+			expectedError:  errors.New("invalid event to exclude: blah*"),
+		},
```

Comment:
First off, thanks for adding tests! Could you add a few test cases for the following cases?

-  having the `*` elsewhere in the filters (i.e. 'bl*ah')
- combinations of filters, in some cases where both valid, or one invalid, or both invalid (i.e. 'event=open*,exec*' or 'event!=open* event=open*'

Reply:
For testing the `event!=<prefix>*` test cases, I couldn't find a succinct way to set `EventsToTrace` to be "every event ID but x".  Since there are over 1000 event IDs didn't think we'd want hard code a long slice with nearly every event ID in it.  Am I thinking about this wrong?

Reply:
Ah, yea that'd be obnoxious. So I think the way this _should_ be done is having a separate test function where instead of having a comparison of each valid event it'd just check the length or that certain ones are contained in the list. 

However, that's way too much to ask of an open source contribution. If you feel particularly motivated to do that (no pressure) you can open up a second PR for it, but otherwise just changing the `strings.Index` -> `len` that I suggested above should be plenty for closing up this PR!

Reply:
Sounds good.  I'll look into it and potentially open a new issue for this.